src/sndfile.h.in

```diff
@@ -116,7 +116,14 @@ enum
 	SF_FORMAT_ALAC_24		= 0x0072,		/* Apple Lossless Audio Codec (24 bit). */
 	SF_FORMAT_ALAC_32		= 0x0073,		/* Apple Lossless Audio Codec (32 bit). */
 
-	/* Endian-ness options. */
+	SF_FORMAT_MP3_STEREO		= 0x0080,		/* MP3 MODE. */
+	SF_FORMAT_MP3_JTSTEREO		= 0x0081,		/* MP3 MODE. */
+	SF_FORMAT_MP3_DUAL		= 0x0082,		/* MP3 MODE. */
+	SF_FORMAT_MP3_MONO		= 0x0083,		/* MP3 MODE. */
+	/* Endian-nes
+
+
+ options. */
```

Comment:
The comment there should be unchanged.

Reply:
Sorry; my bad typing and I did not notice.  fixed.